src/temp/Explore/MapLayers.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, ChangeEvent } from 'react';
+import style from '../Explore/Explore.module.scss';
+import { MapLayerToggle } from './ExploreProject';
+import InfoIcon from '../icons/InfoIcon';
+import { StyledSwitch } from './CustomSwitch';
+import { YearRangeSlider } from './ExploreProject';
+import { useTranslation } from 'next-i18next';
+import { YearRangeSliderProps } from './ExploreProject';
+
+const MapLayers = ({ startYear, endYear }: YearRangeSliderProps) => {
```

Comment:
The name `MapLayers` suggests a layer displayed on the map. Rename to `MayLayerOptions` or `ExploreProjectOptions`